app/src/main/java/org/simple/clinic/login/LoginOtpSmsListener.kt

```diff
@@ -5,4 +5,6 @@ import io.reactivex.Completable
 interface LoginOtpSmsListener {
 
   fun listenForLoginOtp(): Completable
+
+  fun listenForLoginOtpBlocking()
```

Comment:
~For all the blocking calls we are using immediate as suffix. Should we change this to match that, WDYT?~

Reply:
NVM. This has been replaced in later commits by single method `listenForLoginOtp`